Mathlib/SetTheory/ZFC/Basic.lean

```diff
@@ -470,14 +470,14 @@ def sUnion : ZFSet → ZFSet :=
           fun b hb => ⟨b, PSet.Equiv.symm hb⟩⟩
 
 @[inherit_doc]
-prefix:110 "⋃₀ " => ZFSet.sUnion
+scoped prefix:110 "⋃₀ " => ZFSet.sUnion
```

Comment:
Can you update the doc for `ZFSet.sUnion` to mention the notation and the scope?

Reply:
(and for the other three notations)